app.py

```diff
@@ -1392,6 +1395,56 @@ def get_specific_adventure(name, level, mode):
                            ))
 
 
+@app.route('/embedded/<int:level>', methods=['GET'])
+def get_embedded_code_editor(level):
+    forget_current_user()
+
+    # Start with an empty program
+    program = ''
+
+    # If for any reason the level is invalid, set to level 1
+    try:
+        level = int(level)
+        if level < 1 or level > hedy.HEDY_MAX_LEVEL:
+            program = '# Gekozen leven is ongeldig, Hedy staat nu op level 1'
```

Comment:
Should we try to make this a translatable message? Perhaps instead of a comment we can show an alert message to the user somehow? What do you think?